packages/flutter/lib/src/material/tabs.dart

```diff
@@ -636,12 +742,24 @@ class _TabBarState extends State<TabBar> {
   @override
   void didUpdateWidget(TabBar oldWidget) {
     super.didUpdateWidget(oldWidget);
-    if (widget.controller != oldWidget.controller)
+    if (widget.controller != oldWidget.controller ||
+        widget.indicatorColor != oldWidget.indicatorColor ||
```

Comment:
It looks like _updateTabController() short circuits if newController == _controller, so the new or conditions here doesn't do anything no?

Should we have a test that changes one of these parameters (e.g indicatorColor) and verifies that the change worked?